ios/engine/KMEI/KeymanEngine/resources/Keyman.bundle/Contents/Resources/keyboard.html

```diff
@@ -70,6 +71,14 @@
             function getOskHeight() {
                 return oskHeight;
             }
+        
+            function beepKeyboard() {
+                fragmentToggle = (fragmentToggle + 1) % 100;
```

Comment:
Is it better to move `beepKeyboard()` after the declaration of `fragmentToggle`?  (l. 92)

Could JS handle it as `undefined`?